bridge/client/app/_components/ktb-sequence-state-info/ktb-sequence-state-info.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, EventEmitter, Input, Output, ViewEncapsulation } from '@angular/core';
+import {Sequence} from '../../_models/sequence';
+
+@Component({
+  selector: 'ktb-sequence-state-info',
+  templateUrl: './ktb-sequence-state-info.component.html',
+  styleUrls: ['./ktb-sequence-state-info.component.scss'],
+  host: {
+    class: 'ktb-sequence-state-info',
+  },
+  encapsulation: ViewEncapsulation.None,
+})
+export class KtbSequenceStateInfoComponent {
+
+  private _sequence?: Sequence;
+  private _showOnlyLastStage = false;
+
+  @Output() readonly stageClicked = new EventEmitter<{ sequence: Sequence, stage?: string }>();
+
+  @Input()
+  get sequence(): Sequence | undefined {
+    return this._sequence;
+  }
+  set sequence(sequence: Sequence | undefined) {
+    if (this._sequence !== sequence) {
+      this._sequence = sequence;
+    }
+  }
+
+  @Input()
+  get showOnlyLastStage(): boolean {
+    return this._showOnlyLastStage;
+  }
+  set showOnlyLastStage(showOnlyLastStage: boolean) {
+    if (this._showOnlyLastStage !== showOnlyLastStage) {
+      this._showOnlyLastStage = showOnlyLastStage;
+    }
+  }
+
+  constructor() {
+  }
+
+  getStages(): (string | undefined)[] | undefined {
+    return this.showOnlyLastStage ? [this.sequence?.getLastStage()] : this.sequence?.getStages();
```

Comment:
I had a version with the same idea, returning just a `string[]`, but it looked ugly. A `ngIf` in the html Template is much shorter, but since it doesn't change anything in regard to application performance, I'll change it.